src/Uno.UI/UI/Xaml/FrameworkElement.Android.cs

```diff
@@ -84,6 +84,12 @@ protected override void OnNativeUnloaded()
 			}
 		}
 
+
+		partial void OnFlowDirectionChanged(DependencyPropertyChangedEventArgs args)
+		{
+			this.LayoutDirection = FlowDirection == FlowDirection.RightToLeft ? LayoutDirection.Rtl : LayoutDirection.Ltr;
```

Comment:
I'm not hitting this line on Android. I can't see obvious reason. @jeromelaban @MartinZikmund  Can you help here?

Reply:
Maybe this could be the cause - https://github.com/MicrosoftDocs/xamarin-docs/issues/2817#issuecomment-722321951 - setting supportsRtl on application may be required

Reply:
Didn't seem to work :/

Also tried `ViewCompat.SetLayoutDirection(this, FlowDirection == FlowDirection.RightToLeft ? (int)LayoutDirection.Rtl : (int)LayoutDirection.Ltr);`